reference/entry.json

```diff
@@ -771,6 +771,66 @@
         ]
       }
     },
+    "/events/{eventId}/type/{termId}": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/eventId"
+        },
+        {
+          "$ref": "#/components/parameters/termId"
+        }
+      ],
+      "put": {
+        "summary": "Update type",
+        "operationId": "put-event-type",
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "400": {
+            "description": "Bad Request. Possible error types:\n\n* https://api.publiq.be/probs/url/path-parameter-invalid",
+            "content": {
+              "application/problem+json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                },
+                "examples": {
+                  "Example": {
+                    "value": {
+                      "type": "https://api.publiq.be/probs/url/path-parameter-invalid",
+                      "title": "Path parameter invalid",
+                      "status": 400,
+                      "detail": "Term with id \"1.2.1.0.0\" can not be used as an event type."
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "$ref": "#/components/responses/EventNotFound"
+          }
+        },
+        "tags": [
+          "Events"
+        ],
+        "description": "Updates the event's type (examples of types are `Concert`, `Opendeurdag`, `Lessenreeks`, and so on) based on the given `eventId` and `termId`.\n\nTerms are pre-defined and can be found using our [Terms](./docs/Terms.md) guide. Only terms from the `eventtype` domain can be used as event types.\n\nIf the `eventId` does not exist a `404 Not Found` will be returned. If the `typeId` does not exist or is not a term in the `eventtype` domain, a `400 Bad Request` will be returned. If the request is successful a `204 No Content` will be returned.",
```

Comment:
It took a bit of experimentation to fix this, and the path in the code is now a bit weird and it says in the Stoplight Studio editor that the link does not work when you click it, but it now works in the docs.

I think this is because their link resolvers are already a bit weird when they render markdown files / OpenAPI files and it seems like it changed in the "new design" because I just noticed all links in Markdown files to OpenAPI operations in the UiTPAS project are broken as well...